Author: Steven Chamberlain <steven@pyro.eu.org>
Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Description: This patch avoids FTBFSes on any architecture whose "netstat -an"
 output is not what we expect or just wrong, see:
 http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=663010
 NOTE: While this patch at a first sight looks a bit dangerous it isn't that
 bad: if virtuoso couldn't actually start in these make_vad.sh scripts, some 
 *.vad files wouldn't be genereated; these *.vad files are being installed by
 the package, therefore if one of them is missing the package would fail to
 build anyway.
--- a/binsrc/rdf_mappers/make_vad.sh
+++ b/binsrc/rdf_mappers/make_vad.sh
@@ -276,8 +276,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/bpel/make_vad.sh
+++ b/binsrc/bpel/make_vad.sh
@@ -281,8 +281,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/b3s/make_vad.sh
+++ b/binsrc/b3s/make_vad.sh
@@ -231,8 +231,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/tutorial/make_vad.sh
+++ b/binsrc/tutorial/make_vad.sh
@@ -131,8 +131,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/yacutia/mkvad.sh
+++ b/binsrc/yacutia/mkvad.sh
@@ -178,8 +178,8 @@ START_SERVER()
       nows=`expr $nows + $nowh \*  60`
       if test $nows -ge $timeout
         then
-    ECHO "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-    exit 1
+    ECHO "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+    return 1
   fi
   done
 }
--- a/binsrc/samples/demo/mkdoc.sh
+++ b/binsrc/samples/demo/mkdoc.sh
@@ -191,8 +191,8 @@ START_SERVER()
            nows=`expr $nows + $nowh \*  60`
            if test $nows -ge $timeout
            then
-       	ECHO "***FAILED: Could not start Virtuoso DOC Server within $timeout seconds"
-       	exit 1
+       	ECHO "***WARNING: Could not start Virtuoso DOC Server within $timeout seconds"
+       	return 1
            fi
        done
    fi
--- a/binsrc/samples/sparql_demo/make_vad.sh
+++ b/binsrc/samples/sparql_demo/make_vad.sh
@@ -141,8 +141,8 @@ virtuoso_start() {
 	nows=`expr $nows + $nowh \*  60`
 	if test $nows -ge $timeout
 	then
-	    LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-	    exit 1
+	    LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+	    return 1
 	fi
     done
 }
--- a/binsrc/sync/make_vad.sh
+++ b/binsrc/sync/make_vad.sh
@@ -131,8 +131,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/isparql/make_vad.sh
+++ b/binsrc/isparql/make_vad.sh
@@ -144,8 +144,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Addressbook/make_vad.sh
+++ b/appsrc/ODS-Addressbook/make_vad.sh
@@ -120,8 +120,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Bookmark/make_vad.sh
+++ b/appsrc/ODS-Bookmark/make_vad.sh
@@ -124,8 +124,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Briefcase/make_vad.sh
+++ b/appsrc/ODS-Briefcase/make_vad.sh
@@ -140,8 +140,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Calendar/make_vad.sh
+++ b/appsrc/ODS-Calendar/make_vad.sh
@@ -123,8 +123,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Community/make_vad.sh
+++ b/appsrc/ODS-Community/make_vad.sh
@@ -127,8 +127,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Discussion/make_vad.sh
+++ b/appsrc/ODS-Discussion/make_vad.sh
@@ -137,8 +137,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-FeedManager/make_vad.sh
+++ b/appsrc/ODS-FeedManager/make_vad.sh
@@ -123,8 +123,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Framework/make_vad.sh
+++ b/appsrc/ODS-Framework/make_vad.sh
@@ -165,8 +165,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Framework/oauth/make_vad.sh
+++ b/appsrc/ODS-Framework/oauth/make_vad.sh
@@ -215,8 +215,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Gallery/make_vad.sh
+++ b/appsrc/ODS-Gallery/make_vad.sh
@@ -127,8 +127,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Polls/make_vad.sh
+++ b/appsrc/ODS-Polls/make_vad.sh
@@ -122,8 +122,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-WebMail/make_vad.sh
+++ b/appsrc/ODS-WebMail/make_vad.sh
@@ -123,8 +123,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Wiki/make_vad.sh
+++ b/appsrc/ODS-Wiki/make_vad.sh
@@ -158,8 +158,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/hosting/mono/tests/tclrsrv.sh
+++ b/binsrc/hosting/mono/tests/tclrsrv.sh
@@ -241,8 +241,8 @@ START_SERVER ()
           nows=`expr $nows + $nowh \*  60`
           if test $nows -ge $timeout
           then
-              LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-              exit 1
+              LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+              return 1
           fi
         done
 }
--- a/binsrc/samples/demo/mkdemo.sh
+++ b/binsrc/samples/demo/mkdemo.sh
@@ -170,8 +170,8 @@ START_SERVER()
       nows=`expr $nows + $nowh \*  60`
       if test $nows -ge $timeout
         then
-	  ECHO "***FAILED: Could not start Virtuoso DEMO Server within $timeout seconds"
-	  exit 1
+	  ECHO "***WARNING: Could not start Virtuoso DEMO Server within $timeout seconds"
+	  return 1
 	fi
   done
 }
--- a/binsrc/samples/xpath/files2dav.sh
+++ b/binsrc/samples/xpath/files2dav.sh
@@ -121,8 +121,8 @@ START_SERVER()
       nows=`expr $nows + $nowh \*  60`
       if test $nows -ge $timeout 
         then
-	  ECHO "***FAILED: Could not start Virtuoso DEMO Server within $timeout seconds"
-	  exit 1
+	  ECHO "***WARNING: Could not start Virtuoso DEMO Server within $timeout seconds"
+	  return 1
 	fi
   done
 }
--- a/binsrc/samples/xquery/files2dav.sh
+++ b/binsrc/samples/xquery/files2dav.sh
@@ -121,8 +121,8 @@ START_SERVER()
       nows=`expr $nows + $nowh \*  60`
       if test $nows -ge $timeout 
         then
-	  ECHO "***FAILED: Could not start Virtuoso DEMO Server within $timeout seconds"
-	  exit 1
+	  ECHO "***WARNING: Could not start Virtuoso DEMO Server within $timeout seconds"
+	  return 1
 	fi
   done
 }
--- a/binsrc/tests/biftest/thook.sh
+++ b/binsrc/tests/biftest/thook.sh
@@ -154,8 +154,8 @@ START_SERVER()
 	    nows=`expr $nows + $nowh \*  60`
 	    if test $nows -ge $timeout
 	    then
-		LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-		exit 1
+		LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+		return 1
 	    fi
 	done
 }
--- a/binsrc/tests/suite/test_fn.sh
+++ b/binsrc/tests/suite/test_fn.sh
@@ -333,8 +333,8 @@ START_SERVER()
 	    nows=`expr $nows + $nowh \*  60`
 	    if test $nows -ge $timeout
 	    then
-		LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-		exit 1
+		LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+		return 1
 	    fi
 	done
 }
--- a/binsrc/vsp/admin/debug/make_vad.sh
+++ b/binsrc/vsp/admin/debug/make_vad.sh
@@ -97,8 +97,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/vspx/suite/vspx_suite.sh
+++ b/binsrc/vspx/suite/vspx_suite.sh
@@ -66,8 +66,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start Virtuoso Server within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start Virtuoso Server within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/appsrc/ODS-Blog/make_vad.sh
+++ b/appsrc/ODS-Blog/make_vad.sh
@@ -139,8 +139,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
--- a/binsrc/samples/demo/make_vad.sh
+++ b/binsrc/samples/demo/make_vad.sh
@@ -142,8 +142,8 @@ virtuoso_start() {
     nows=`expr $nows + $nowh \*  60`
     if test $nows -ge $timeout
     then
-      LOG "***FAILED: Could not start $SERVER within $timeout seconds"
-      exit 1
+      LOG "***WARNING: Could not start $SERVER within $timeout seconds"
+      return 1
     fi
   done
 }
